packages/select/src/Select.tsx

```diff
@@ -496,59 +497,63 @@ export default function Select({
 
   return (
     <div className={cx(wrapperStyle, className)}>
-      {label && (
-        <Label
-          htmlFor={menuButtonId}
-          id={labelId}
-          darkMode={darkMode}
-          disabled={disabled}
-          className={cx(
-            {
-              [largeLabelStyles]: size === Size.Large,
-              [css`
-                font-size: ${baseFontSize}px;
-                line-height: 20px;
-              `]: size === Size.Default,
-            },
-            css`
-              // Prevent hover state from showing when hovering label
-              pointer-events: none;
-            `,
-            css`
-              ${MobileMediaQuery} {
-                font-size: ${mobileSizeSet.label.text}px;
-                line-height: ${mobileSizeSet.label.lineHeight}px;
-              }
-            `,
+      {(label || description) && (
+        <div className={labelDescriptionContainerStyle}>
+          {label && (
+            <Label
+              htmlFor={menuButtonId}
+              id={labelId}
+              darkMode={darkMode}
+              disabled={disabled}
+              className={cx(
+                {
+                  [largeLabelStyles]: size === Size.Large,
+                  [css`
+                    font-size: ${baseFontSize}px;
+                    line-height: 20px;
+                  `]: size === Size.Default,
```

Comment:
Feels like these style blocks should be in a `.styles.tsx` file with the other size variants considered

Reply:
I'm going to leave this as is for now since all I needed to do was add a wrapper around this block of code. This component needs a bigger refactor which is not in scope.